exec/java-exec/src/test/java/org/apache/drill/exec/store/log/TestLogReader.java

```diff
@@ -781,9 +781,19 @@ public void testPluginSerialization() throws IOException {
 
   @Test
   public void testFirewallSchema() throws RpcException {
-    String sql = "SELECT * FROM cp.`regex/firewall.ssdlog`";
+    String sql = "SELECT * FROM cp.`regex/firewall.ssdlog` limit 0";
     RowSet result = client.queryBuilder().sql(sql).rowSet();
-    result.print();
+    TupleMetadata expectedSchema = new SchemaBuilder()
+        .add("eventDate", TypeProtos.MinorType.TIMESTAMP, TypeProtos.DataMode.OPTIONAL)
```

Comment:
Simpler: `addNullable("eventDate", MinorType.TIMESTAMP)`

Import `MinorType`, use the `addNullable` method rather than the more verbose form of using the `DataMode`. That is, express intent, not implementation.